pytorch_lightning/trainer/connectors/logger_connector/epoch_result_store.py

```diff
@@ -97,11 +97,11 @@ def get_reduced_metrics(self):
         return self._internals_reduced
 
     def add_dataloader_idx(self):
```

Comment:
about the names is it an action `add_dataloader_idx` but the content is more check if any was added...